packages/salesforcedx-vscode-core/src/commands/forceApexClassCreate.ts

```diff
@@ -56,7 +57,12 @@ class ForceApexClassCreateExecutor extends SfdxCommandletExecutor<
     }).execute(cancellationToken);
 
     execution.processExitSubject.subscribe(async data => {
-      this.logMetric(execution.command.logName, startTime);
+      const dirType = response.data.outputdir.endsWith(
```

Comment:
These template commands badly need to be refactored to avoid having so much duplicated code. I started doing so but stuck it in another branch to separate it from this work. I'll make a story for it.